hieradata/clients/private.vpn.jenkins.io.yaml

```
@@ -1,37 +1,18 @@
 ---
 profile::openvpn::networks:
   eth0:
+    # Defined in https://github.com/jenkins-infra/azure-net/blob/main/vnets.tf
     name: Subnet 'app-tier' of network 'public-vnet'
     route-metric: 100
-    # macaddress: 00:0d:3a:0e:4e:7e
-    # network_cidr: 10.0.1.0/24
     network_cidr: 10.244.0.0/14
-  # eth0:
-  #   # name: Subnet 'dmz' of network 'prod-jenkins-public-prod'
-  #   name: Subnet 'dmz' of network 'prod-jenkins-public-prod'
-  #   route-metric: 100
-  #   macaddress: 00:0d:3a:0e:4b:1c
-  #   # network_cidr: 10.0.99.0/24
-  #   network_cidr: 10.0.99.0/24
-  # eth1:
-  #   # name: Subnet 'data-tier' of network 'prod-jenkins-public-prod'
-  #   name: Subnet 'data-tier' of network 'prod-jenkins-public-prod'
-  #   route-metric: 200
-  #   macaddress: 00:0d:3a:0e:4f:89
-  #   # network_cidr: 10.0.2.0/24
-  #   network_cidr: 10.0.2.0/24
-  #   peered_network_cidrs:
-  #     # # Network 'prod-jenkins-private-prod-vnet' through peering
-  #     # Network 'prod-jenkins-private-prod-vnet' through peering
-  #     # - 10.240.0.0/14
-  #     - 10.240.0.0/14
-  # eth2:
-    # name: Subnet 'app-tier' of network 'prod-jenkins-public-prod'
-    # name: Subnet 'app-tier' of network 'public-vnet'
-    # route-metric: 300
-    # macaddress: 00:0d:3a:0e:4e:7e
-    # # network_cidr: 10.0.1.0/24
-    # network_cidr: 10.244.0.0/14
+  eth1:
+    # Defined in https://github.com/jenkins-infra/azure/blob/main/privatek8s.tf
+    name: Subnet 'privatek8s-tier' of network 'private-vnet'
+    route-metric: 200
+    network_cidr: 10.249.0.0/16
+    peered_network_cidrs:
+      # Network 'private-vnet' through peering
+      - 10.248.0.0/14
```

```suggestion
    # Defined in https://github.com/jenkins-infra/azure/blob/main/privatek8s.tf
    name: Network 'private-vnet'
    route-metric: 200
    network_cidr: 10.248.0.0/14      
```

After carefully reviwing https://github.com/lemeurherve/jenkins-infra/blob/c3eeb9d2328a4c0778b58fbdd1366c80cf9266c9/dist/profile/manifests/openvpn.pp#L127-L146, it appears that the `network_cidr` is only used for the routing table, not for the NIC configuration.

It means we can simplify by specifying the whole network containint the `eth1`'s NIC subnet

---

```suggestion
    # Defined in https://github.com/jenkins-infra/azure-net/blob/main/vnet.tf
    name: 'private-vnet'
    route-metric: 200
    network_cidr: 10.248.0.0/14
```